registry/insecure.md

```diff
@@ -129,7 +129,8 @@ certificate at the OS level.
 #### Ubuntu
 
 ```bash
-$ cp certs/domain.crt /usr/local/share/ca-certificates/myregistrydomain.com.crt
+$ mkdir -p /etc/docker/certs.d/myregistrydomain.com
+$ cp certs/domain.crt /etc/docker/certs.d/myregistrydomain.com
```

Comment:
I don't think this directory is specific to the Docker daemon and not for the OS level.

http://manpages.ubuntu.com/manpages/precise/man8/update-ca-certificates.8.html

Reply:
There are two ways to get an engine to trust a registry.  The first is adding a root cert to the system trust store, which is what is currently documented in registry/insecure.md. 

The second way is to add it to /etc/docker/certs.d/myregistrydomain.com.  There is no need to do an "update-ca-certs" type of step.  And it might not even be necessary to restart the docker engine (I can't remember).  The location is different on windows/mac. 

The down side of the second way is that it only affects the engine.  You'd also have to tell the CLI about the cert for when it contacts the registry directly (docker trust, docker manifest).